aea/cli/interact.py

```diff
@@ -166,7 +172,7 @@ def _try_construct_envelope(
         )
         msg.counterparty = agent_name
         msg.sender = sender
-        assert dialogues.update(msg) is not None
+        enforce(dialogues.update(msg) is not None, "Dialogue was not created.")
```

Comment:
This is one of those cases where the replacement of assert is actually crucial. The line would not be evaluated on the LHS if debug flag is not set! Our enforce will always be evaluated!